ansible/roles/candlepin/tasks/el7/el7_base_deps.yml

```diff
@@ -0,0 +1,93 @@
+---
+- name: "Disable SELinux"
```

Comment:
If we're adding in support for el9 in this branch, we can also drop el7 support and not need any specific workarounds for it anymore.

Reply:
ack